### ISSUE TITLE
ACE notes "serious" error for missing a11y metadata in EPUB3

### ISSUE THREAD

**Issue by jbrugge** [daisy/pipeline#697]
### Expected Behavior
When running the dtbook-to-epub3 script on DAISY books, the EPUB3 result should have an OPF file with the required accessibility metadata properties that ACE designates as "serious" when missing:
* metadata-accessmode
* metadata-accessibilityfeature
* metadata-accessibilityhazard

### Actual Behavior
Running ACE against the EPUB3 output generates a report pointing out the missing properties.
`  "a11y-metadata": {
    "missing": [
      "schema:accessMode",
      "schema:accessibilityFeature",
      "schema:accessibilityHazard",
      "schema:accessibilitySummary",
      "schema:accessModeSufficient",
      "a11y:certifiedBy",
      "a11y:certifierCredential",
      "a11y:certifierReport",
      "dcterms:conformsTo"
    ],
    "empty": [],
    "present": []
  },
`

### Steps to Reproduce
The sample book in #674 exhibits this issue, so refer to the EPUB3 output from that.

The ACE report was then generated with this:
`$ ace --subdir --silent --outdir ace_sherlock The_Adventures_of_Sherlock_H.epub`

### Environment
* Operating system: Linux 64-bit
* DAISY Pipeline 2 version: 1.14.7
* Interface: web service API
* ACE: 1.3.1

### Logs

[ACE JSON report]( https://gist.github.com/jbrugge/bc5bef0952f6c1b03a5b2f97ef7a7473 )

**Comment by bertfrees** [comment]
Fixed by https://github.com/daisy/pipeline-modules/commit/efb1b61b38395b33a5d0a942d27a761d8ffd95c4
